packages/components/package.json

```
@@ -190,5 +190,10 @@
       "@babel/plugin-proposal-object-rest-spread",
       "dev-expression"
     ]
-  }
+  },
+  "sideEffects": [
+    "es/bundle.js",
+    "es/globals/js/boot.js",
+    "es/globals/js/watch.js"
```

Sorry for not telling `umd` directory:

```suggestion
    "es/bundle.js",
    "es/globals/js/boot.js",
    "es/globals/js/watch.js",
    "umd/bundle.js",
    "umd/globals/js/boot.js",
    "umd/globals/js/watch.js"
```